sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/RequestResponseChannel.java

```diff
@@ -155,7 +162,8 @@ protected RequestResponseChannel(String connectionId, String fullyQualifiedNames
                 receiveLink.open();
             });
         } catch (IOException e) {
-            throw logger.logExceptionAsError(new RuntimeException("Unable to open send and receive link.", e));
+            throw logger.logExceptionAsError(new RuntimeException(String.format(
```

Comment:
Would changing this to a `Mono.error` return be a breaking change? Or does something further up the stack handle converting this into a `Mono.error`?

Reply:
It wouldn't be a breaking change. It would be propagated back up.